test/intrinsics.jl

```diff
@@ -164,8 +164,17 @@ end
     @test_intrinsic Core.Intrinsics.fptoui UInt Float16(3.3) UInt(3)
 end
 
-@test Core.Intrinsics.atomic_fence(:sequentially_consistent) === nothing
+using Base.Experimental: @force_compile
```

Comment:
I had to remove this from 1.7 (since it doesn't have `@force_compile`). We'll see if the tests pass without it.